.github/actions/js-lint/action.yml

```diff
@@ -0,0 +1,29 @@
+# Define action name, description, and inputs
+name: "JS Lint"
+description: "Lint code with Prettier and eslint"
+inputs:
+  working-directory:
+    required: false
+    description: "Working directory"
+    default: "."
+
+# Define action steps
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/setup-node@v2
+      with:
+        node-version: 16.x
+    - run: |
+        yarn
+        yarn prettier --check .
+      shell: bash
```

Comment:
Let's split the job into separate "run" keys, there's not benefit in keeping this in one (unlike Dockerfile where each layer is cached and costs disk space) and it's easier to catch which step failed.